### PR TITLE
add test metrics to CLI tests

### PR DESCRIPTION
**Pull request by nsinkov**
## Changes proposed in this PR

- add test metrics to CLI tests

### CODE REVIEW

File: cli/integration/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                result = 'unknown'
+            metrics = {
+                'build-id': os.getenv('TEST_METRICS_BUILD_ID', None),
+                'expected-to-fail': xfail_mark is not None and xfail_mark.name == 'xfail',
+                'git-branch': git_branch,
+                'git-branch-under-test': os.getenv('TEST_METRICS_BRANCH_UNDER_TEST', None),
+                'git-commit-hash': git_commit_hex,
+                'git-commit-hash-under-test': os.getenv('TEST_METRICS_COMMIT_HASH_UNDER_TEST', None),
+                'host': socket.gethostname(),
+                'project': 'waiter',
```
Comment (dposada) [inline cli/integration/tests/conftest.py]: Maybe `waiter-cli`?

File: cli/integration/tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                'result': result,
+                'run-attempt': os.getenv('TEST_METRICS_RUN_ATTEMPT', None),
+                'run-description': os.getenv('TEST_METRICS_RUN_DESCRIPTION', 'open source integration tests'),
```
Comment (dposada) [inline cli/integration/tests/conftest.py]: Maybe `open source CLI integration tests`?

Comment (nsinkov) [reply cli/integration/tests/conftest.py]: ok

File: cli/integration/tests/conftest.py (hunk 1 above)
Comment (nsinkov) [reply cli/integration/tests/conftest.py]: ok

**Pull request merged**